packages/text-annotator/src/state/TextAnnotationStore.ts

```diff
@@ -15,7 +15,9 @@ export interface TextAnnotationStore<T extends TextAnnotation = TextAnnotation>
 
   getAnnotationRects(id: string): DOMRect[];
 
-  getAt(x: number, y: number, filter?: Filter): T | undefined;
+  getAt(x: number, y: number, all: true, filter?: Filter): T[] | undefined;
+  getAt(x: number, y: number, all: false, filter?: Filter): T | undefined;
+  getAt(x: number, y: number, all?: boolean, filter?: Filter): T | T[] | undefined;
```

Comment:
Should we use the `options: { all?: boolean, filter?: Filter }` type argument to avoid passing an unnamed boolean args like that: `store.getAt(event.clientX - x, event.clientY - y, false, currentFilter);`?